lib/govuk_app_config/govuk_prometheus_exporter.rb

```diff
@@ -14,6 +14,7 @@ def self.should_configure
   def self.configure
     return unless should_configure
 
+    require "sidekiq/api"
```

Comment:
Heh, this is a slightly tricky one because there's a ton of iffy dynamic loading stuff going on here :)

This library needs to work with apps that don't use Sidekiq as well as ones that do. So if we're trying to fix an undeclared conditional dependency of `prometheus_exporter` on `sidekiq/api` I think we'd need to put the conditional inside the `if defined?(Sidekiq)` below. i.e. just underneath what's currently line 22.

Reply:
I tried that but it didn't work. I think it needs to be required before `require "prometheus_exporter"` 🤔

Reply:
Maybe we should go for `require "sidekiq/api" if defined?(Sidekiq)` on line 17?

Reply:
That's ok but slightly less desirable, because you're duplicating the condition so there's just one more thing to go wrong as the file changes over time.

Reply:
I just tried it again and it worked in that if block. _(My terminal history confirms I tired it before. Maybe I haven't restarted the worker... anyway.)_ I'll pop it in before `require "prometheus_exporter/instrumentation"` so they are all nicely together. That is much better – thank you!

Reply:
No worries!